app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<section class="container flex food-placeholder">
+
+<div class="food flex">
+    <h1 class = "txt-center"> Recipes</h1>
+    <div class = "btn-right">
+        <%= button_to "Create new recipe", 
+                new_user_recipe_path,
+                method: :get, 
+                class: "btn-right" %>
+    </div>
+
+<% if @recipes.any? %>
+        <% @recipes.each do |recipe| %>
+            <div class='recipe flex wide-gap'>
+            <div class="recipe-details flex">
+                     <%= link_to recipe.name, 
+                     user_recipe_path(recipe.user_id, recipe),
+                        method: :get %> 
+                        <% if can? :destroy, recipe %>
+                            <%= button_to "remove", 
+                                user_recipe_path(current_user, recipe),
+                                method: :delete %> 
+                        <% end %>  
+                            
+                
+           </div>
+           <p><%= recipe.description %></p>
+           </div> 
+           
+        <% end %>
+    </div>    
+<% else %>
+    <p> There are no recipes yet. </p>   
+<% end %>
+</div>
+</section>
```

Comment:
_The first reviewer does not mention it. I highly recommend fixing it._

**OPTIONAL**
- I create two users for the test, the first recipe in the table is from the old user. When ı click on it ı am getting this page. it is nice to show some errors. But;

- [ ] Users shouldn't see other users' recipe list, they should only have their list. Kindly fix it.

![Screenshot from 2023-02-20 16-59-21](https://user-images.githubusercontent.com/38635158/220128466-604617fa-0f42-4223-86f0-10ae3d71a623.png)